docs/tutorials/services/accessing-and-moving-robot-arm.md

```diff
@@ -45,39 +45,38 @@ Before starting this tutorial, make sure you have the [Viam Python SDK](https://
 
 If you are connecting to a real robotic arm during this tutorial, make sure your computer can communicate with the controller before continuing.
 
-## Configure a robot
+## Configure a machine
 
 1. Navigate to the [machine page on the Viam app](https://app.viam.com/robots).
-2. Create a new machine.
-3. Follow the instructions on the **Setup** tab.
-4. Select the **Config** tab.
-5. Under the **Components** subtab, click **Create component** in the lower-left corner and create a component with the following attributes:
+2. Add a new machine.
+3. Click on the machine and then click on **View setup instructions**.
+   Then follow the instructions to set up your machine part until your machine is connected successfully.
+4. Once your machine is live, select the **CONFIGURE** tab.
+5. Click on the **+** symbol next to your machine in the **Builder** panel and select **Component** in the menu that opens:
 
-   - Choose `Arm` as the type.
+   - Choose `arm` as the type.
    - Choose your desired model.
-     - If you're using an xArm 6, choose the `xArm6` model from the dropdown list.
+     - For example, if you're using an xArm 6, choose the `xArm6` model from the list.
    - Enter `myArm` as the **Name** for this component, then click **Create**.
 
-6. In the newly created `myArm` component panel, fill in some additional details:
+6. In the newly created `myArm` component panel, configure any attributes as needed.
+   Refer to the documentation for the model for information about your arm's model.
 
-   - Enter the IP address for your robot controller in the **Host** field
-   - Enter a desired velocity _(in degrees per second)_ for the robot arm's motions in the **Speed** field
-     - For an `xArm6`, a safe value is `15` degrees per second.
+7. Switch to the **Frame** panel to see the default values for your arm.
+   You do not need to change the default values that populate the new frame card.
 
-7. Add a **Frame** to this component.
+   <!-- TODO -->
 
-   - You do not need to change the default values that populate the new frame card
+   {{<imgproc src="/tutorials/motion/access_02_arm_config.png" resize="1000x" declaredimensions=true alt="Sample machine arm configuration with several fields filled out." style="max-width: 600px" >}}
 
-   {{<imgproc src="/tutorials/motion/access_02_arm_config.png" resize="1000x" declaredimensions=true alt="Sample robot arm configuration with several fields filled out." style="max-width: 600px" >}}
+8. Save your machine configuration.
 
-8. Save this robot configuration.
-
-9. Go to the **Code sample** tab and select the programming language you are working in.
+9. Go to the **CONNECT** tab and select the programming language you are working in.
```

Comment:
```suggestion
9. Go to the **Code sample** page of the **CONNECT** tab and select the programming language you are working in.
```
(opt) Not 100% necessary as they will land here automatically but just in case people get confused I have been being specific like this